projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -594,15 +595,29 @@ export const childBlocks = [
 		name: JetpackFieldSingleChoice.name,
 		settings: mergeSettings( FieldDefaults, {
 			...JetpackFieldSingleChoice.settings,
-			deprecated: [ multiFieldV1( 'radio' ) ],
+			deprecated: [
+				{
+					save() {
+						return <InnerBlocks.Content />;
+					},
+				},
+				multiFieldV1( 'radio' ),
+			],
 		} ),
 	},
 	JetpackFieldSingleChoiceItem,
 	{
 		name: JetpackFieldMultipleChoice.name,
 		settings: mergeSettings( FieldDefaults, {
 			...JetpackFieldMultipleChoice.settings,
-			deprecated: [ multiFieldV1( 'checkbox' ) ],
+			deprecated: [
+				{
+					save() {
+						return <InnerBlocks.Content />;
+					},
+				},
```

Comment:
For anyone like me following the history of deprecations for Jetpack Forms fields, the lack of attributes and block support definitions here appear to have been an oversight only. 

cc/ @enejb - it's best practice to always define the attributes and block supports for a deprecation. The [docs](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) actually show that only the `migrate` and `isEligible` properties on a deprecation are optional. Defining the attributes and supports in the deprecation freezes them as they were at the time of that legacy version of the block and prevents the deprecation breaking as the block evolves in the future.

FWIW I'll fix up these deprecations when migrating fields to using inner blocks.